Revised Pathpoint docs for V2 re-launch (#17972)
newrelic/docs-website: src/content/docs/new-relic-solutions/business-observability/intro-pathpoint.mdx

freshnessValidatedDate: never
---

import addStages from 'images/pathpoint_v2_screenshot_add_stages.webp'

import addSteps from 'images/pathpoint_v2_screenshot_add_steps.webp'

import addSignals from 'images/pathpoint_v2_screenshot_add_signals.webp'

import flowKPIs from 'images/pathpoint_v2_screenshot_flow_kpis.webp'

import playback from 'images/pathpoint_v2_screenshot_playback.webp'

import stages from 'images/pathpoint_v2_screenshot_stages.webp'

import steps from 'images/pathpoint_v2_screenshot_steps.webp'

import signals from 'images/pathpoint_v2_screenshot_signals.webp'

import signalDetails from 'images/pathpoint_v2_signal_details.webp'

import overview from 'images/business-observability_diagram_NR-biz-obs-overview.webp'

> New Relic Pathpoint is an enterprise platform tracker that offers a unique approach to business process observability. It models system health in relation to actual business and user-impacting concepts specific to your digital business.

Reviewer: ```suggestion
New Relic Pathpoint is an enterprise platform tracker that offers a unique approach to business process observability. It models system health in relation to actual business and user-impacted concepts specific to your digital business.
```

## Overview of Pathpoint resources [#overview-docs]

The sections below cover important concepts and procedures about Pathpoint.

## Pathpoint elements [#elements]

> Pathpoint can be a key part of optimizing your business observability. It maps an organization's business process or "customer journey" to tangible signals from various services, alerts, and infrastructure to help you rapidly pinpoint where your business or customer journey is impacted.  This top down approach to observability allows you share observability data with a wide variety of technical and non-technical personnel.

Reviewer: ```suggestion
Pathpoint can be a key part of optimizing your business observability. It maps an organization's business process or "customer journey" to tangible signals from various services, alerts, and infrastructure to help you rapidly pinpoint where your business or customer journey is impacted. This top down approach to observability allows you to share observability data with a wide variety of technical and non-technical personnel.
```

## Overview of Pathpoint resources [#overview-docs]

> This set of docs will tell you important concepts and procedures about Pathpoint.

Reviewer: ```suggestion
The sections below cover important concepts and procedures about Pathpoint.
```

The different stages of a Pathpoint enables you to gain a high-level overview of business processes and identify the different services and methods presented at the system level for each commercial stage.

### Adding stages [#adding-stages]
    
Adding stages is as fast as typing in the stage names.  

    <img
        title="Add stages"
        alt="Add stages"
        src={addSteps}
    />

> ### Get real-time and historical information for each stage [#real-time-info-for-stage]

Reviewer: ```suggestion
### Get historical and real-time information for each stage [#real-time-info-for-stage]
```

> You can view real-time information related to a particular stage including any errors associated with it. You can see if any of the associated signals for a particular stage are critical, in which case the stage will turn yellow. If everything is working normally, the stage will stay green. If all signals have anomalies that need to be resolved, the stage will turn red.

Reviewer: ```suggestion
View real-time details for each stage, including any errors. Stages are color-coded to indicate their status: green for normal operation, yellow for critical signals, and red for multiple anomalies requiring attention.
```

        src={steps}
    />

> Steps break down stages but also offer a level of abstraction. Often steps can be represented in some sequential order, known as levels. This allows New Relic to intuitively understand if there is some sequential dependencies between steps.

Reviewer: ```suggestion
Steps break down stages but also offer a level of abstraction. Often steps can be represented in some sequential order, known as levels. This allows New Relic to intuitively understand if there are some sequential dependencies between steps.
```

### Adding steps [#adding-steps]

    <img
        title="Add steps"
        alt="Add steps"
        src={addSteps}
    />

</Collapser>
<Collapser id="pathpoint-signals" title="Signals">

Signals are the most granular elements in the Pathpoint model. They are often a specific application monitored by our <InlinePopover type="apm" /> or <InlinePopover type="browser" />, but you can use any New Relic entity or alert as a signal. This provides almost limitless flexibility.

Every signal provides a snapshot of its current status, connected information, and a record of previous states.

    <img
        title="Pathpoint signals"
        alt="Pathpoint signals"
        src={signals}
    />

### Adding signals [#adding-signals]

> Adding signals to Pathpoint is as simple as navigating a list of entities and alerts in your account.  It is easy to filter them by name and add only the ones you want.  No custom queries are required.

Reviewer: ```suggestion
Adding signals to Pathpoint is as simple as navigating a list of entities and alerts in your account. It is easy to filter them by name and add only the ones you want. No custom queries are required.
```

        src={steps}
    />

> Steps breakdown stages but offer a level of abstraction.  Often steps can be represented in some sequential order (known as levels).  This allows us to intuitively understand if there is some sequential dependencies between steps.

Reviewer: ```suggestion
Steps breakdown stages but offer a level of abstraction. Often steps can be represented in some sequential order (known as levels). This allows us to intuitively understand if there is some sequential dependencies between steps.
```

### Adding steps [#adding-steps]

    <img
        title="Add steps"
        alt="Add steps"
        src={addSteps}
    />

</Collapser>
<Collapser id="pathpoint-signals" title="Signals">

> Signals are the most granular elements in the Pathpoint model. They are often a specific application monitored by our <InlinePopover type="apm" /> or <InlinePopover type="browser" />, but you can use any New Relic entity or a new relic alert as a signal.  This provides almost limitless flexibility.

Reviewer: ```suggestion
Signals are the most granular elements in the Pathpoint model. They are often a specific application monitored by our <InlinePopover type="apm" /> or <InlinePopover type="browser" />, but you can use any New Relic entity or a new relic alert as a signal. This provides almost limitless flexibility.
```

> Each signal encapsulates a while of information such as the current status of that signal, information about related entities, and the historical knowledge of previous states.

Reviewer: ```suggestion
Every signal provides a snapshot of its current status, connected information, and a record of previous states.
```

    <img
        title="Pathpoint signals"
        alt="Pathpoint signals"
        src={signals}
    />

### Adding signals [#adding-signals]

> Adding signals to Pathpoint is as simple as navigating a list of entities and alerts in your account.  It is easy to filter them by name and add only the ones you want.  No custom queries are required.

Reviewer: ```suggestion
Easily add signals to Pathpoint by selecting from a list of entities and alerts within your account. Filter by name to find the exact signals you need — no custom queries required.
```

    <img
        title="Add signals"
        alt="Add signals"
        src={addSignals}
    />

### Signal status [#touchpoint-status]

> Like the entities or alerts they are related to signals have three different health statuses, which are color-coded for easy identification:

Reviewer: ```suggestion
Signals, like the entities and alerts they're linked to, have three health states indicated by color:
```

> * Green: The signal is healthy, and everything is working as expected.
> * Gray: The signal requires attention.  Perhaps there is a configuration issue (such as a missing alert) or the telemetry is not available.
> * Red: The signal is in a fail state, and there are critical issues that require immediate attention.
> 
> Every step in Pathpoint is connected to one or more signals. As such, the health status of every step is dependent on the status of its respective signals. Therefore, signals are considered the pulse of each Pathpoint. By monitoring the status of signals, you can gain insights into the health of the entire system and identify any anomalies that need attention.

Reviewer: ```suggestion
* Green: Optimal performance. All systems are operating as expected.
* Gray: Requires attention. Potential configuration issues or data gaps exist, such as a missing alert or the telemetry is not available.
* Red: Critical failure. Immediate action is necessary.

Each Pathpoint step relies on one or more signals, making signal health crucial to overall Pathpoint performance. By tracking signal status, you can effectively monitor system health and quickly identify issues.
```

<Callout variant="tip">
    By default, you'll only see all signals. However you can choose to see "unhealthy only" or "critical only" to simplify your view.
</Callout>

### View signal details [#view-signal-details]

> Signals that are directly tied to a New Relic entity will inherit the status of that entity.  Clicking on a signal icon will pop up a window that shows you all of the relevant information about your signal such as:

Reviewer: ```suggestion
Signals linked to a New Relic entity adopt the entity's health status. Clicking a signal icon displays a window with detailed signal information, including:
```

> 1. Any open incidents
> 2. The status of any golden signals
> 3. A link to that entities details page to drill down further

Reviewer: ```suggestion
* Open incidents
* Golden signal status
* Link to the entity's details page for deeper analysis
```

    <img
        title="View signal details"
        alt="View signal details"
        src={signalDetails}
    />

The image above shows the details page for an APM entity signal.

</Collapser>

<Collapser id="business-flow-kpis" title="Business flow KPIs"> 

> In addition to the structured signals that are always associated with a New Relic entity or alert, you can add a flexible KPI using a regular NRQL query.  These allow you to keep an eye on overall business impacts that are related to your flow while trying to understand the health of your lower level entities.  Some customers want to have counts for important critical events like user logins, user registrations, successful checkouts, carts abandoned as well as important dollar amounts like total checkout value, total returned product value.   You can usually find some of these in your existing telemetry and they don’t have to be associated with an entity.  They can be logs, custom metrics, etc.

Reviewer: ```suggestion
Beyond structured signals tied to New Relic entities or alerts, Pathpoint supports flexible KPI tracking via custom NRQL queries. This enables correlation of lower-level entity health with broader business impact metrics. Common use cases include monitoring event counts (such as user logins, registrations, checkouts, cart abandonment, etc.) and monetary values (such as total checkout value, returned product value, etc.). These KPIs can be derived from existing telemetry sources such as logs, custom metrics, or other data independent of specific entities.
```
